ui/src/ensembler/list/ListEnsemblersTable.js

```diff
@@ -0,0 +1,187 @@
+import React, { Fragment, useMemo } from "react";
+import {
+  EuiBadge,
+  EuiBasicTable,
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiSearchBar,
+  EuiSpacer,
+  EuiText,
+  EuiToolTip
+} from "@elastic/eui";
+import { appConfig } from "../../config";
+import { EnsemblerType } from "../../services/ensembler/EnsemblerType";
+import moment from "moment";
+import { FormLabelWithToolTip } from "../../components/form/label_with_tooltip/FormLabelWithToolTip";
+
+const { defaultTextSize, defaultIconSize, dateFormat } = appConfig.tables;
+
+export const ListEnsemblersTable = ({
+  items,
+  totalItemCount,
+  isLoaded,
+  error,
+  page,
+  filter,
+  onQueryChange,
+  onPaginationChange,
+  onRowClick,
+  ...props
+}) => {
+  const searchQuery = useMemo(() => {
+    const parts = [];
+    if (!!filter.search) {
+      parts.push(filter.search);
+    }
+    if (!!filter.type) {
+      parts.push(`type:${filter.type}`);
```

Comment:
1. The motivation is simple: if the API supports pagination, then you need to support filtering at the back-end as well, because otherwise your search will only be limited to the results shown on a single page (which is not how I imagine search should work). 
Routers API doesn't support pagination (i.e. all results are fetched at once and then pagination is only added on the client-side for the users convenience), that's why `EuiInMemoryTable` is used there with a client-side search/filtering. 

I don't think that back-end pagination matters that much for the Ensemblers View (I won't expect that user will have more than handful of ensemblers inside a single project), but I expect that it's definitely required for the Jobs View, hence I decided to standardize the implementation.

2. Do you mean some search-through functionality, where user can find different entities (router, ensembler, experiment etc,) in a single search result? I don't think that it's important for now.

I think that filtering (i.e. filter by status or by ensembler id) is more important than search, but I've added the later just in case.